brainstormforce/ultimate-addons-for-gutenberg
# Fix: CF7-styler - Submit button font size and color issue.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/bbe97de8-ac62-4d3f-b014-a93adfa63951?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop CF7 styler
- Go to style > Submit >Typo > font size and check it is working in the editor or not
- Same for color Add hover color to submit button add bg color adjust the opacity and you will see it is applying in the editor.

### Checklist:
